public/src/components/channelManagement/campaigns/CampaignsSidebar.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { makeStyles, Typography, Button } from '@material-ui/core';
+import { Campaigns, Campaign } from './CampaignsForm';
+import NewCampaignButton from './NewCampaignButton';
+import CampaignsList from './CampaignsList';
+import { red } from '@material-ui/core/colors';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    paddingLeft: '32px',
+  },
+  listsContainer: {
+    position: 'relative',
+    display: 'flex',
+    marginTop: '8px',
+  },
+  buttonsContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '8px',
+    marginBottom: '10px',
+  },
+  saveAllButton: {
+    justifyContent: 'start',
+    height: '48px',
+  },
+  saveAllButtonAlert: {
+    justifyContent: 'start',
+    height: '48px',
+    border: `3px solid ${red[500]}`,
+  },
+  saveAllButtonText: {
+    fontSize: '12px',
+    fontWeight: 500,
+    textTransform: 'uppercase',
+    letterSpacing: '1px',
+  },
+}));
+
+interface CampaignsSidebarProps {
+  campaigns: Campaigns;
+  selectedCampaign?: Campaign;
+  newCampaignCreated: boolean;
+  createCampaign: (campaign: Campaign) => void;
+  onCampaignSelected: (campaignName: string) => void;
+  saveAllCampaigns: () => void;
+}
+
+function CampaignsSidebar({
+  campaigns,
+  createCampaign,
+  selectedCampaign,
+  newCampaignCreated,
+  onCampaignSelected,
+  saveAllCampaigns,
+}: CampaignsSidebarProps): React.ReactElement {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.buttonsContainer}>
+        <NewCampaignButton
+          existingNames={campaigns.map(c => c.name)}
+          existingNicknames={campaigns.map(c => c.nickname)}
```

Comment:
Elsewhere we default to empty string if nickname is undefined (a bit hacky), see https://github.com/guardian/support-admin-console/blob/main/public/src/components/channelManagement/testsForm.tsx#L288
This currently breaks in CODE where nickname isn't defined in the json